core-bundle/contao/modules/ModuleLogin.php

```diff
@@ -203,6 +203,15 @@ protected function compile()
 			return;
 		}
 
+		if (($objLostPasswordTarget = $this->objModel->getRelated('pwResetJumpTo')) instanceof PageModel)
+		{
+			/** @var PageModel $objLostPasswordTarget */
+			$this->Template->passwordreset = $objLostPasswordTarget->getFrontendUrl();
+
+			$this->Template->resetTitle = $GLOBALS['TL_LANG']['MSC']['lostPassword'];
+			$this->Template->resetLabel = $GLOBALS['TL_LANG']['MSC']['lostPassword'];
```

Comment:
any reason to have the same language variable in the template twice? Also, couldn't we just output that directly in the template? 
`<?= $this->trans('MSC.lostPassword', [], 'contao_default'); ?>` 

Not sure we're doing that anywhere, but I personally think the template should be responsible for the view data. @contao/developers any objections?

Reply:
Shouldn't we change every translation within the LoginModule then?
Thus, we could remove the TL_LANG Superglobals one by one, however the translations within overridden templates won't exist anymore.